UnitTests/tUnificationWithAbstraction.cpp

```diff
@@ -74,16 +75,16 @@ TermList int_plus(TermList n1, TermList n2)
 {
   return binary(Theory::INT_PLUS,n1,n2);
 }
-Literal* equals(TermList t1, TermList t2)
+Literal* equals(TermList t1, SortType t2)
```

Comment:
The `SortType` here looks like a mistake

Reply:
Agreed, I have reverted to `TermList`